### PR TITLE
[bitnami/kafka] Added variable to enable/disable auto creation of topics.

### PR DESCRIPTION
**Pull request by adityajalkhare**
Scope of change: Enables toggling of `auto.create.topics.enable` Kafka configuration property

Limitations: No observed limitations as of now. 

Tests: I have tested this in Development environment Kafka Cluster.

**Description of the change**

Now we can set  `auto.create.topics.enable` to `true` or `false` based on our requirement. This was available while running Kafka through `docker run` command but I was unable to set this property while deploying through helm.

**Benefits**

Kafka Administrators/Developers can set this property based on their architectural and functional requirements.

**Possible drawbacks**

Not observed as it's a normal property

**Applicable issues**

Not Found any issues open related to this

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/kafka/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kafka
-version: 9.0.6
+version: 9.1.6
```
Comment (carrodher) [inline bitnami/kafka/Chart.yaml:3]:
```suggestion
version: 9.1.0
```

File: bitnami/kafka/values.yaml [hunk 2]
```diff
@@ -116,6 +116,11 @@ heapOpts: -Xmx1024m -Xms1024m
 ##
 deleteTopicEnable: false
 
+## Switch to enable auto creation of topics.
+## Enabling auto creation of topics not recommended for production or similar environments.
+##
+autoCreateTopicsEnable: false
```
Comment (carrodher) [inline bitnami/kafka/values.yaml:122]:
```suggestion
autoCreateTopicsEnable: true
```

I agree not enabling the auto-creation of topics for production environment but yes for testing purposes. In that case, I would enable that feature in the `values.yaml` and disable it in the `values-production.yaml`.

In that case, can you also add the difference in this [README section](https://github.com/bitnami/charts/tree/master/bitnami/kafka#production-configuration-and-horizontal-scaling)?

Comment (adityajalkhare) [reply bitnami/kafka/values.yaml]: I have edited `values.yaml` as you suggested, and updated [README section](https://github.com/bitnami/charts/tree/master/bitnami/kafka#production-configuration-and-horizontal-scaling) too

**Comment by carrodher** [comment]
Everything fine, above commit from `bitnami-bot` is the result of test this PR in our pipeline, if everything works as expected, container images and dependencies are updated to the latest version.

Now your changes are merged in GitHub and published into the bitnami helm repository.

**Pull request merged**